feature-libs/cart/saved-cart/styles/_index.scss

```diff
@@ -1,7 +1,3 @@
 // TODO: speak to michal about it
 @import './components/index';
 @import 'saved-cart';
-@import 'saved-cart-list';
```

Comment:
let's talk about this as mentioned in the comment above as to why you created a component folder and as to why you're using the white listing.

This feature is quite small and I believe we can just leave it out here as shown in the examples that I provided before